ClusterDisruptionIT.testAckedIndexing failure

Failure in 7.x https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.x+matrix-java-periodic/ES_RUNTIME_JAVA=java11,nodes=general-purpose/548/console

```
11:26:04   2> REPRODUCE WITH: ./gradlew ':server:integTest' --tests "org.elasticsearch.discovery.ClusterDisruptionIT.testAckedIndexing" -Dtests.seed=A1089782D65D0286 -Dtests.security.manager=true -Dtests.locale=ar-SD -Dtests.timezone=America/Punta_Arenas -Dcompiler.java=13
11:26:04   2> java.lang.AssertionError: ClusterRerouteResponse failed - not acked
11:26:04     Expected: <true>
11:26:04          but: was <false>
11:26:04         at __randomizedtesting.SeedInfo.seed([A1089782D65D0286:2BC923718A20E4CD]:0)
11:26:04         at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
11:26:04         at org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcked(ElasticsearchAssertions.java:115)
11:26:04         at org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcked(ElasticsearchAssertions.java:103)
11:26:04         at org.elasticsearch.discovery.ClusterDisruptionIT.testAckedIndexing(ClusterDisruptionIT.java:230)
```

I cannot find any other failures of this test recently. It fails in a different spot then before in #41068 so might be unrelated.

--- a/server/src/test/java/org/elasticsearch/discovery/ClusterDisruptionIT.java
+++ b/server/src/test/java/org/elasticsearch/discovery/ClusterDisruptionIT.java
@@ -228,7 +228,7 @@ public class ClusterDisruptionIT extends AbstractDisruptionTestCase {
                 // is the super-connected node and recovery source and target are on opposite sides of the bridge
                 if (disruptionScheme instanceof NetworkDisruption &&
                     ((NetworkDisruption) disruptionScheme).getDisruptedLinks() instanceof Bridge) {
-                    assertAcked(client().admin().cluster().prepareReroute().setRetryFailed(true));
+                    assertBusy(() -> assertAcked(client().admin().cluster().prepareReroute().setRetryFailed(true)));
                 }
                 ensureGreen("test");
 

